config/routes.rb

```diff
@@ -1,6 +1,19 @@
 Rails.application.routes.draw do
+  devise_for :users, path: '', path_names: {
+    sign_in: 'login', sign_out: 'logout',
+    password: 'secret', confirmation: 'verification',
+    registration: 'register', edit: 'edit/profile'
+  }, sign_out_via: [:get, :post]
+
+  resources :groups, path: 'categories' do
+    resources :expenses
+  end
```

Comment:
- [ ] It would be nice to use only and except options to add only necessary routes in case maybe there are some routes that are not used. 👍🏻

Reply:
```rb
Rails.application.routes.draw do
  devise_for :users, path: '', path_names: {
    sign_in: 'login', sign_out: 'logout',
    password: 'secret', confirmation: 'verification',
    registration: 'register', edit: 'edit/profile'
  }, sign_out_via: [:get, :post]

  resources :groups, path: 'categories', only: [:new, :create, :index, :show, :destroy, :edit] do
    resources :expenses, only: [:new, :create, :index, :show, :destroy]
  end
  root to: "splash#index"
end
```